ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -78,6 +78,18 @@ tests:
     - chain: ipi-install-heterogeneous-day2-kerneltype
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-byo-iam-role-arm-f28
```

Comment:
If we covered `aws-ipi-byo-iam-role-fips-amd`, then do not need to cover `aws-ipi-byo-iam-role-arm`, `fips` is only supported on amd arch. so the two install profile is almost the same.

Reply:
Let's remove aws-ipi-byo-iam-role-arm

Reply:
Hi Jianlin, from install/e2e perspective, does it make sense to keep aws-ipi-byo-iam-role-fips-amd only?

Reply:
Either `aws-ipi-byo-iam-role-arm` or `aws-ipi-byo-iam-role-fips-amd` okay for me, but we just need one of them. About `fips` coverage, refer to https://github.com/openshift/release/pull/54740#discussion_r1699351848

Reply:
Ah, okay. I thought both are present in e2e. Checked 4.16 e2e jobs, we have aws-ipi-byo-iam-role-arm-f2 and aws-ipi-byo-iam-role-fips-amd-f9-destructive. Makes sense.